### DISCUSSION TITLE
dificulty signing up student devoloper pack ( no edu mail )

### DISCUSSION THREAD

**Discussion by mustafizz007** [community/community#64192] [GitHub Education]
### Hi everyone,

trying to signup using my university id card but it seems like without an edu mail, i cant be getting any further of the promt edu mail required.

**Comment by jadstrike** [comment]
I suggest try again by providing some voucher containing date related to the university. But I'm not sure if you can get without edu mail. Discuss with your university officer if possible. Good Luck.
